### PR TITLE
Event parameter is optional in logging

### PR DESCRIPTION
**Pull request by donnapep**
Removing required event parameter check since not every table will have an event field.

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review. Thx.

**Comment by rodrigopavezi** [comment]
@donnapep looks good :+1: Curious, why are we not logging throttle events? cheers

**Comment by donnapep** [comment]
@rodrigopavezi We needed to decrease the frequency of some requests being logged numerous times per second and eventually crashing the Presentation. Logging throttle events would not resolve that problem.

File: src/js/logger.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ RiseVision.Common.Logger = (function(utils) {
    *  Public Methods
    */
   function log(tableName, params) {
-    if (!tableName || !params || !params.event || isThrottled(params.event)) {
+    if (!tableName || !params || (params.event && isThrottled(params.event))) {
```
Comment (stulees) [inline src/js/logger.js]: @donnapep Won't this allow a falsy (undefined, null, etc) `params.event` value pass by removing `!params.event ||`?

Comment (donnapep) [inline src/js/logger.js]: @stulees Can you check again? Does that change address your concern?

**Comment by stulees** [comment]
@donnapep :+1:

**Pull request merged**